### ISSUE TITLE
Error in tldr git

### ISSUE THREAD

**Issue by mcchrish** [tldr-pages/tldr-node-client#17]
This only happens in git, the rest of the command seems ok.
```
tldr git                                                                                                                                                              
/usr/local/lib/node_modules/tldr/node_modules/marked/lib/marked.js:1226
    throw e;
          ^
TypeError: Cannot set property 'code' of undefined
Please report this to https://github.com/chjj/marked.
    at Renderer.r.codespan (/usr/local/lib/node_modules/tldr/lib/parser.js:47:32)
    at InlineLexer.output (/usr/local/lib/node_modules/tldr/node_modules/marked/lib/marked.js:664:28)
    at Parser.tok (/usr/local/lib/node_modules/tldr/node_modules/marked/lib/marked.js:1075:50)
    at Parser.tok (/usr/local/lib/node_modules/tldr/node_modules/marked/lib/marked.js:1033:22)
    at Parser.parse (/usr/local/lib/node_modules/tldr/node_modules/marked/lib/marked.js:935:17)
    at Function.Parser.parse (/usr/local/lib/node_modules/tldr/node_modules/marked/lib/marked.js:922:17)
    at marked (/usr/local/lib/node_modules/tldr/node_modules/marked/lib/marked.js:1218:19)
    at Object.exports.parse (/usr/local/lib/node_modules/tldr/lib/parser.js:50:3)
    at /usr/local/lib/node_modules/tldr/lib/tldr.js:89:23
    at /usr/local/lib/node_modules/tldr/lib/cache.js:22:5
```

**Comment by rprieto** [comment]
Thanks for reporting this @mcchrish, sounds like an issue processing a particular Markdown item.

**Comment by igorshubovych** [comment]
We have the same issue in tldr-pages repo:
https://github.com/tldr-pages/tldr/issues/340

The problem happened because of backticks in command description

~~~Markdown
# git

> Main command for all `git` commands
~~~

I fixed it in page, but frankly speaking I think client should be more robust.
